tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator/ExtensionsMetadataGenerator.csproj

```diff
@@ -21,17 +31,24 @@
 
   <Target Name="PackReferenceAssemblies" AfterTargets="Build">
     <ItemGroup>
-      <None Include="$(OutputPath)\*.dll">
+      <Content Include="$(OutputPath)\*.dll">
         <Pack>true</Pack>
         <PackagePath>tools\netstandard2.0\</PackagePath>
+      </Content>
+      <None Include="$(OutputPath)\generator\*">
+        <Pack>true</Pack>
+        <PackagePath>tools\netstandard2.0\generator</PackagePath>
       </None>
     </ItemGroup>
   </Target>
   
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Framework" Version="15.3.409" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.3.409" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.0.2" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.0.2" >
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
@fabiocav -- had to set this as PrivateAssets so they didn't show up as dependencies. Do you know if you were doing that manually previously?